docs/development/deploying-locally/index.md

```diff
@@ -18,62 +19,45 @@ $ source ~/.virtualenvs/reana/bin/activate
+(reana) $ reana-dev cluster-build
 ```
 
 **5.** Deploy REANA:
 
 ```console
-(reana) $ make deploy
+(reana) $ reana-dev cluster-deploy --admin-email john.doe@example.org --admin-password 123456
 ```
 
 **6.** Run an example:
 
 ```console
-(reana) $ DEMO=reana-demo-helloworld make example
+(reana) $ reana-dev run-example -c reana-demo-helloworld
 ```
 
-## Makefile variables reference
-
-| Variable              | Description                                                                                         | Default value  |
-|-----------------------|-----------------------------------------------------------------------------------------------------|----------------|
-| `CLUSTER_FLAGS`       | Which [values](https://github.com/reanahub/reana/blob/master/helm/reana/README.md) need to be passed to `helm install`? (e.g. `debug.enabled=true,ui.enable=true`) | - |
-| `DEMO`                | Which [demo example](https://github.com/reanahub?q=reana-demo) to run? (e.g. [`reana-demo-helloworld`](https://github.com/reanahub/reana-demo-helloworld)) | several |
-| `EXCLUDE_COMPONENTS`  | Which REANA components should be excluded from the build? [e.g. `reana-ui,r-m-broker`]                | -              |
-| `GITHUB_USER`         | Which GitHub user account to use for cloning the REANA repositories?                               | `anonymous`    |
-| `INSTANCE_NAME`       | Which name/prefix to use for your REANA instance?                                                   | `reana`        |
-| `MINIKUBE_CPUS`       | How many CPUs to allocate for Minikube?                                                             | `2`            |
-| `MINIKUBE_DISKSIZE`   | How much disk size to allocate for Minikube?                                                        | `40g`          |
-| `MINIKUBE_DRIVER`     | Which [VM driver](https://minikube.sigs.k8s.io/docs/reference/drivers/) to use for Minikube?        | `virtualbox`   |
-| `MINIKUBE_KUBERNETES` | Which [Kubernetes version](https://github.com/kubernetes/kubernetes/releases) to use with Minikube? | `v1.16.3`      |
-| `MINIKUBE_MEMORY`     | How much memory to allocate for Minikube?                                                           | `3072`         |
-| `SERVER_URL`          | Setting a customized REANA Server hostname? (e.g. `https://example.org`)                            | `https://`[`` `minikube ip` ``](https://minikube.sigs.k8s.io/docs/reference/commands/ip/) |
-| `TIMECHECK`           | Checking frequency in seconds when bringing cluster up and down?                                    | `5`            |
-| `TIMEOUT`             | Maximum timeout to wait when bringing cluster up and down?                                          | `300`          |
-
 ## Examples
 
 REANA can be deployed in development mode with code reload and [debugging capabilities](../debugging):
 
 ```console
-(reana) $ CLUSTER_FLAGS="debug.enabled=true" make deploy
+(reana) $ reana-dev cluster-deploy --mode debug
```

Comment:
We need to create the kind cluster and build the REANA images in debug mode too to make the debugging mode work:
```diff
+(reana) $ reana-dev cluster-create --mode debug
+(reana) $ reana-dev cluster-build --mode debug
(reana) $ reana-dev cluster-deploy --mode debug
```

Reply:
Done ✅